src/sass/base/_b-utils.scss

```diff
@@ -1,3 +1,7 @@
 .full-width {
   width: 100%;
 }
+
+.no-background-image {
```

Comment:
Won't there still be left side padding here? I think making an alert specific no icon class would be good (i.e. a `usa-alert-no-icon` class that fixed the padding and hide the bg image).